packetbeat/docs/fields.asciidoc

```diff
@@ -10049,7 +10096,7 @@ type: keyword
 
 --
 
-*`tls.server_certificate_chain`*::
+*`tls.detailed.server_certificate_chain`*::
```

Comment:
how is this different than `tls.server.certificate_chain`? I see the code referencing both.